lib/rubocop/cop/rails/not_null_column.rb

```diff
@@ -0,0 +1,44 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for add_column call with NOT NULL constraint
+      # in migration file.
+      #
+      # @example
+      #   # bad
+      #   add_column :users, :name, :string, null: false
+      #
+      #   # good
+      #   add_column :users, :name, :string, null: true
+      class NotNullColumn < Cop
+        MSG = 'Do not add a NOT NULL column'.freeze
```

Comment:
:+1: when I first scanned through this PR I thought this was a cop that simply encouraged people to not have `NOT NULL` columns